research/mfac/README.md

```diff
@@ -0,0 +1,113 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Matrix-Free Approximate Curvature (M-FAC) Pruning
+
+The paper
+[Efficient Matrix-Free Approximations of Second-Order Information, With Applications to Pruning and Optimization](https://arxiv.org/pdf/2107.03356.pdf)
```

Comment:
Since "With" can be capped or not depending on which writing style is used, looking at the paper, they've lowercased it to be "with" I think.